### DISCUSSION TITLE
Will Laravel echo and laravel-echo-server continue to support Socket.io?

### DISCUSSION THREAD

**Discussion by stratboy** [laravel/framework#48993] [Q&A]
I was wondering, because it feels strange that socket.io, which is quite the only really free js tool for websockets, is never mentioned in Laravel docs. But still, in Laravel Echo source code I can find some reference to it. So, is it still supported, in its v4? And will it be supported in the future? 
Or does Laravel only (will) support commercial websocket libraries??

**Comment by antimech** [comment]
`laravel-echo-server` had no updates for a year, so, most likely no. But don't worry there are better alternatives now:
https://laravel.com/docs/10.x/broadcasting#open-source-alternatives
